contents/testing/security-testing.md

```diff
@@ -0,0 +1,93 @@
+<frontmatter>
+  title: Introduction to Security Testing
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Security Testing
+
+**Author(s): [Ahmed Bahajjaj](https://github.com/madanalogy)**<br>
+Reviewers: [James Pang](https://github.com/jamessspanggg)
+
+<box id="article-toc">
+
+* [Introduction to Security Testing](#introduction-to-security-testing)
+    * [What Is Security Testing?‎](#what-is-security-testing)
+    * [Why Have Security Testing?](#why-have-security-testing)
+    * [How to Get Started with Security Testing?](#how-to-get-started-with-security-testing)
+    * [Additional Resources‎](#additional-resources)
+</box>
+
+## What Is Security Testing?
+
+Security Testing is a type of Software Testing that uncovers vulnerabilities, threats, risks in a software application and prevents malicious attacks from intruders. 
+
+The goal of security testing is to:
+- Identify threats in the system.
+- To measure the potential vulnerabilities of the system.
+- To help in detecting every possible security risks in the system.
+- To help developers in fixing the security problems through coding.
+
+### Types of Security Testing
+- **Vulnerability Scanning**: This is done through automated software to scan a system against known vulnerability signatures.
+- **Security Scanning**: It involves identifying network and system weaknesses, and later provides solutions for reducing these risks. This scanning can be performed both manually and automatically.
+- **Penetration Testing**: This kind of testing simulates an attack from a malicious hacker. This testing involves analysis of a particular system to check for potential vulnerabilities to an external hacking attempt.
+- **Risk Assessment**: This testing involves analysis of security risks observed in the organization. Risks are classified as  Low, Medium and High. This testing recommends controls and measures to reduce the risk.
+- **Security Auditing**: This is an internal inspection of Applications and Operating systems for security flaws. An audit can also be done via line by line inspection of code.
+- **Posture Assessment**: This combines Security Scanning, Penetration Testing, and Risk Assessments to show an overall security posture of an organization.
```

Comment:
This list is a bit wobbly too as they don't seem to be strictly the same type of thing. If you took the list from some authoritative source, citing the source can give confidence to the reader.